COMETwebapp/Pages/Viewer/ViewerBase.cs

```diff
@@ -164,6 +164,14 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
                     this.UpdateTreeUI(node);
                 };
 
+                this.SessionAnchor.OnSessionRefreshed += (sender, args) =>
+                {

```

Comment:
it seems like what all views will do is repopulate, that is **not** the intent. Lets discuss.